sdk/src/client.ts

```diff
@@ -965,6 +972,70 @@ export class VaultClient {
         return lpToken.getAccountInfo(this.vaultState.fees.referralFeeReceiver);
     }
 
+    /**
+     * Creates LP ATA creation and deposit ix transactions for Realms (https://realms.today/)
+     * @param reserveTokenOwner DAO's wallet
+     * @param amount amount
+     * @param userReserveTokenAccount DAO's reserve token account
+     * @param lpTokenAccountFeePayer The proposal creator that pays for the LP ATA creation
+     * @returns
+     */
+    async realmsDepositIxs({
```

Comment:
If this is for realms only, why have it in the sdk?

Reply:
True, no need for this specific of a method inside the class